supabase/migrations/20260816201458_harden_regenerate_hashed_apikey_rpc.sql

```
@@ -0,0 +1,91 @@
+  RETURN public.regenerate_hashed_apikey_for_user(p_apikey_id, v_user_id);
+END;
+$$;
+
+ALTER FUNCTION public.regenerate_hashed_apikey(bigint) OWNER TO postgres;
+REVOKE ALL ON FUNCTION public.regenerate_hashed_apikey(bigint) FROM PUBLIC;
+GRANT ALL ON FUNCTION public.regenerate_hashed_apikey(bigint) TO service_role;
+GRANT ALL ON FUNCTION public.regenerate_hashed_apikey(bigint) TO anon;
+GRANT ALL ON FUNCTION public.regenerate_hashed_apikey(bigint) TO authenticated;
+
+COMMENT ON FUNCTION public.regenerate_hashed_apikey(bigint) IS
+  'Public compatibility RPC for hashed API key rotation. Resolves the caller from JWT or capgkey, requires org.manage_apikeys on every org the target key is bound to, then returns plaintext via the service-owned helper. Granted to anon+authenticated so capgkey PostgREST can call it; fail-closed unless the caller has manage_apikeys so a read-only sibling key cannot steal privileges (GHSA-4h9w-86j7-q2p3).';
```

SQLFluff (4.3.0)

[error] Line is too long (408 > 80).

(LT05)